### PR TITLE
Create stub for CoKleisli.agda

### PR DESCRIPTION
**Pull request by tetrapharmakon**
The associativity property is a game of `assoc`, `refl⟩∘⟨` and `⟩∘⟨refl`... More expert hands will fill the hole!

### CODE REVIEW

File: agda-categories.agda-lib [hunk 1]
```diff
@@ -1,3 +1,3 @@
 name: agda-categories
-depend: standard-library-1.7
+depend: standard-library-2.0
```
[the remark this answers is not in the document]
Comment (tetrapharmakon) [reply agda-categories.agda-lib:2]: My version is 2.0; the version here 1.7.
Is there a way to accept all the changes apart the one to this file?

**Review by tetrapharmakon — COMMENTED** [review]
With this commit I did all I wanted to in this branch (add duals of all files `Keisli.agda`). Is there anything more to do? @JacquesCarette

File: agda-categories.agda-lib (hunk 1 above)
[the remark this answers is not in the document]
Comment (JacquesCarette) [reply agda-categories.agda-lib:2]: No, the change to the file will have to be explicitly taken out of the PR.

**Comment by JacquesCarette** [comment]
This is looking excellent. I'll figure out out to use stdlib-1.7 and merge this in.

**Pull request merged**